src/Sentry.Unity.Android/AndroidJavaScopeObserver.cs

```diff
@@ -8,22 +6,16 @@ namespace Sentry.Unity.Android
     /// Scope Observer for Android through Java (JNI).
     /// </summary>
     /// <see href="https://github.com/getsentry/sentry-java"/>
-    public class AndroidJavaScopeObserver : IScopeObserver
+    public class AndroidJavaScopeObserver : ScopeObserver
     {
-        private readonly SentryOptions _options;
-
-        public AndroidJavaScopeObserver(SentryOptions options) => _options = options;
+        public AndroidJavaScopeObserver(SentryOptions options) : base("Android", options) { }
 
+        // TODO can we use a local var `AndroidJavaClass _sentryJava` initialized in a constructor instead?
```

Comment:
I believe the instance can't be shared across threads. I tried on my first attempt and learned with some errors, if my memory doesn't fail me. Also, we need to lazily instantiate things when possible to add less overhead during "init", so if possible to cache the reference, we could, but not in the ctor in this case

```suggestion
```